erc-hl-nicks.el

```diff
@@ -344,6 +344,13 @@
                   (erc-button-add-face (car bounds) (cdr bounds)
                                        (erc-hl-nicks-make-face trimmed)))))))))))
 
+(defun thblt/erc-hl-nicks-refresh-colors ()
```

Comment:
Could you remove the `thblt/` prefix?

Reply:
Err, sorry for the mistake, this comes from my init.el. Should be good now.

Reply:
Oh, no worries!  Thanks for the PR.